idea-plugin/src/main/resources/localization/MetaIdeBundle_en.properties

```diff
@@ -0,0 +1,3 @@
+example.text=Test example
```

Comment:
Is there a need to have both ? MetaIdeBundle_en and the other if both are in english

Reply:
No, there is no need of both! still trying things out.. but my guess is that _en wont be needed and default file will be the one without any suffix

Reply:
Let's confirm this and use it that way since otherwise we'll end up with duplicated files for no reason. We also want to be sure so we can even remove alternative translations for stuff we don't want to translate like feature paths for example, that people always refers to by their English names: "Type proofs" vs "Pruebas de tipos".